app/pods/settings/users/index/route.js

```diff
@@ -23,14 +22,11 @@ export default Route.extend(DataTableRouteMixin, {
     }
 
     return options;
-  },
+  }
 
-  // TODO: octane-refactor
-  // eslint-disable-next-line ember/no-actions-hash
-  actions: {
-    refresh() {
-      this._super(...arguments);
-      this.refresh();
-    },
-  },
-});
+  @action
+  refreshRoute() {
+    this.super(...arguments);
+    this.refresh();
```

Comment:
Falende test lijkt mij met deze aanpassing te maken te hebben. Kan je hier `super.refresh()` van maken?

Reply:
Issue lag bij de calls naar `this.send('refreshRoute')` vanuit de controller. Ik dacht dat die `.send()` een methode was om Route methodes op te roepen, maar dat is dus om `action`s op te roepen, zowel uit de route als de controller. Aangezien de controller ook een `refreshRoute` actie had kwamen er recursie-related errors tevoorschijn.